### DISCUSSION TITLE
Migrate to New Passwords module

### DISCUSSION THREAD

**Discussion by antcorro** [YetiForceCompany/YetiForceCRM#15538] [How to] [answered]
Hello,
I have installed version 6.2 service pack 4 and security fix 2.
Now I can upgrade to v6.3 **but** in the comment I see:  "The current Passwords module is outdated and will be removed in the next version. Please consider migrating your data to the new passwords module."

How can I migrate data from old Passwords module to new one?

I have to migrate date before or after upgrading to v6.3?
I don't want to lose datas.

Thanks

Regards
Antonio

**Comment by rskrzypczak** [comment, marked as answer]
> I have to migrate date before or after upgrading to v6.3?

After. Data migration is only possible in version 6.3.
1. Update to version 6.3
2. Export the passwords to the file from the old module.
    ![image](https://user-images.githubusercontent.com/10221999/145720287-2bed8cc2-5c8b-4b53-9d80-e2aeca54206b.png)
3. Import the passwords in the new module.
4. Read - How to use passwords in the new module ([link](https://yetiforce.com/en/knowledge-base/documentation/user-documentation/item/passwords-module)).

**Reply by antcorro** [reply]
Thank you very much
